Add release notes for the epic version of Metals :D (#1278)
scalameta/metals: docs/build-tools/gradle.md

## Manual installation

In a highly customized workspaces it might not be possible to use automatic

> import. In such cases it is quite simple to add the capability to generate the

Reviewer: ```suggestion
import. In such cases it's quite simple to add the capability to generate the
```

Author: Fixed

This will enable us to work with Metals and all features should work.

> 

Reviewer: It's so nice to see all of this go 🎉
